src/DynamoSandbox/Program.cs

```diff
@@ -62,6 +63,8 @@ public string CommonDataRootFolder
 
     internal class Program
     {
+        private static readonly SettingsMigrationWindow migrationWindow = new SettingsMigrationWindow();
```

Comment:
This will be created regardless of migration actually happening, can we `new` only in `SettingsMigrationEventArgs.EventStatusType.Begin`?